rpc/cloudagent/compute/virtualmachine/moc_cloudagent_virtualmachine.proto

```diff
@@ -92,6 +92,10 @@ message HardwareConfiguration {
 	DynamicMemoryConfiguration dynamicMemoryConfiguration = 3;
 }
 
+message GuestAgentConfiguration {
```

Comment:
Can we move the common configuration to https://github.com/microsoft/moc/blob/main/rpc/common/moc_common_computecommon.proto

Reply:
Moved to common compute configuration